src/checkout-buttons/strategies/paypal-commerce/paypal-commerce-button-strategy.ts

```diff
@@ -48,24 +51,34 @@ export default class PaypalCommerceButtonStrategy implements CheckoutButtonStrat
     }
 
     deinitialize(): Promise<void> {
+        this.providerId = undefined;
+
         return Promise.resolve();
     }
 
-    private async _setupPayment(methodId: string, cartId: string): Promise<string> {
-        const { orderId } = await this._paypalCommerceRequestSender.setupPayment(methodId, cartId);
+    private _handleClickButtonProvider(providerId: string, { fundingSource }: ClickDataOptions): void {
+        this.providerId = fundingSource === 'credit' ? providerId + 'credit' : providerId;
```

Comment:
@helga1507 what is the default value for `providerId `? I think for now it would be more obvious to replace `providerId + 'credit'` -> `paypalcommercecredit` as long as this provider Id is the only we one case we want to handle

Reply:
Okay. Thank you!